apps/coscrad-frontend/src/configurable-front-matter/data/configurable-content-schema.ts

```diff
@@ -17,6 +17,7 @@ export type ConfigurableContent<T extends CategorizableType = CategorizableType>
     songIdToCredits: Record<string, string>;
     videoIdToCredits: Record<string, string>;
     shouldEnableWebOfKnowledgeForResources: boolean;
+    listenLive: string;
```

Comment:
I'm not sure I like `listenLive` for this property name. What about `liveStreamPageConfiguration` or `liveStreamInfo` or something like that?

Reply:
let's leave this one for now